### PR TITLE
Build script

### PR DESCRIPTION
**Pull request by briantrice**
I used the WireViz example build script as a template. Should I update the README with the output in the repository?

### CODE REVIEW

**Comment by atomicdog** [comment]
> I used the WireViz example build script as a template. Should I update the README with the output in the repository?

Thanks for the PR.
Do you mean adding the generated pictures into the README file?

**Comment by briantrice** [comment]
> > I used the WireViz example build script as a template. Should I update the README with the output in the repository?
> 
> Thanks for the PR. Do you mean adding the generated pictures into the README file?

Yes; making a Markdown file very similar to the examples directory READMEs in WireViz: https://github.com/wireviz/WireViz/blob/master/examples/readme.md

**Comment by briantrice** [comment]
FYI I just like to make sure I can run a build if I come back to a project after a few months. And it would be nice if we had the outputs to present to people who find YAML a little hard to follow.

**Comment by atomicdog** [comment]
> > > I used the WireViz example build script as a template. Should I update the README with the output in the repository?
> > 
> > 
> > Thanks for the PR. Do you mean adding the generated pictures into the README file?
> 
> Yes; making a Markdown file very similar to the examples directory READMEs in WireViz: https://github.com/wireviz/WireViz/blob/master/examples/readme.md

Yes, that would be cool.

**Comment by atomicdog** [comment]
I've been procrastinating on getting the pictures added, but that was the plan. It still needs to be organized with consistent nomenclature.

**Comment by briantrice** [comment]
So, what I can do is put them in a file named `outputs.md` and you can just curate the elements into a real document. The examples tooling is too automated to bother with.
